code/modules/research/research.dm

```diff
@@ -258,6 +264,7 @@ research holder datum.
 	var/level = 1						//A simple number scale of the research level. Level 0 = Secret tech.
 	var/max_level = 1          // Maximum level this can be at (for job objectives)
 	var/rare = 1						//How much CentCom wants to get that tech. Used in supply shuttle tech cost calculation.
+	var/icon = null						//Name of the FontAwesome icon to represent the tech
```

Comment:
rename this to `ui_icon` or something similar please. `icon` is a native BYOND variable for /atom's and it's probably best that we use a unique var name. Also use proper autodoc format here -> `///`